tutorials/how_to/elmo.md

```diff
@@ -149,5 +149,14 @@ representations = elmo(character_ids)
 
 ## Writing contextual representations to disk
```

Comment:
Is it possible to replace any of the code in the usage example in this tutorial (under `Using ELMo programmatically`) with a link to the new `ElmoEmbedder` class?  Much of the functionality is duplicated, e.g. the `batch_to_ids` function now exits as a method on that class.

Reply:
Cool--I did not know that.  I just copied it from the script.

Reply:
Done.